config/application.rb

```diff
@@ -38,6 +38,7 @@ class Application < Rails::Application
     config.generators.system_tests = nil
 
     # Set default i18n translation to norwegian
+    config.i18n.available_locales = %i[nb en]
```

Comment:
This was added so tests can be run on :en, which helps avoid "missing translation" errors.